rootfs-builder/rootfs.sh

```diff
@@ -222,12 +235,23 @@ if [ -n "${USE_DOCKER}" ] ; then
 	# fake mapping if KERNEL_MODULES_DIR is unset
 	kernel_mod_dir=${KERNEL_MODULES_DIR:-${ROOTFS_DIR}}
 
+	docker_run_args=""
+	docker_run_args+=" --rm"
+	docker_run_args+=" --runtime runc"
+
+	admin_caps=$(distro_needs_admin_caps "$distro")
+	if [ "$admin_caps" = "true" ]; then
+		docker_run_args+=" --cap-add NET_ADMIN"
```

Comment:
why do you need NET_ADMIN? Can you add some comments why these capabilities are required.